core/db/default/spree/states.rb

```diff
@@ -1,14 +1,24 @@
 # frozen_string_literal: true
 
+countries_that_use_nested_subregions = %w(IT)
```

Comment:
If we use this in two places it's probably better to move it into a shared space like a preference?

Reply:
Yeah, I couldn't really think of a good shared spot to put this that made sense. Preferences seems like a weird spot for it IMO, but maybe it's the place that makes the most sense!

Reply:
Yeah, preferences seems like the best place for it after review! I've added a preference in, how does that look?